src/Raven.Server/Documents/Handlers/Admin/Processors/Indexes/AdminIndexHandlerProcessorForTestIndex.cs

```diff
@@ -18,6 +19,10 @@ namespace Raven.Server.Documents.Handlers.Admin.Processors.Indexes;
 
 internal sealed class AdminIndexHandlerProcessorForTestIndex : AbstractAdminIndexHandlerProcessorForTestIndex<DatabaseRequestHandler, DocumentsOperationContext>
 {
+    private static readonly Regex FromIndexClauseRegex = new Regex(
+        @"from index\s+(['""])(.*?)\1",
```

Comment:
Who is generating queries to be validated here?
What if I have: `from      index` ? Or line break in the middle, etc?

Reply:
It comes from the request parameters.
I should add here \s instead of a space - done